src/platform/utilities/api/index.js

```diff
@@ -23,10 +28,16 @@ export function fetchAndUpdateSessionExpiration(url, settings) {
     ...settings,
     async retryOn(attempt, error, response) {
       if (error) return false;
-      const atError = await response.clone().json();
+
+      const errorResponse = isJson
+        ? await response
+            ?.clone()
+            ?.json()
+            .catch(() => response.text())
+        : response.text();
```

Comment:
You might need to clone this as well, I'm seeing an issue with the stream having already been read if not